packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.base.tsx

```diff
@@ -349,6 +353,7 @@ export const CalendarDayGridBase = React.forwardRef(
       weekCorners,
       getDayInfosInRangeOfDay,
       getRefsFromDayInfos,
+      markedDays,
```

Comment:
Please have you considered to add this `shouldMarkDay` prop to `IDay` from `date-time-utilities` and then you won't need to pass anything to CalendarGridRow and have a check in `useWeeks` instead?

Reply:
sure I hadn't considered that but I can make that change